### PR TITLE
Refactor Leds

### PR DESCRIPTION
**Pull request by Aradig26**

### CODE REVIEW

File: src/main/java/frc/robot/commands/intakerelease/WantConeCMD.java [hunk 1]
```diff
@@ -3,22 +3,18 @@
...
+  public WantConeCMD(IntakeRelease intakerelease) {
     this.intakerelease = intakerelease;
-    this.ledStrip = ledStrip;
     addRequirements(intakerelease);
   }
 
   @Override
   public void initialize() {
-    ledStrip.setCmdColorScheme(ColorScheme.WANT_CONE);
+    intakerelease.setWants(Wants.CONE);
```
Comment (john-petrangelo) [inline src/main/java/frc/robot/commands/intakerelease/WantConeCMD.java:17]: Is this change intentional? It's not just an LED change.

File: src/main/java/frc/robot/commands/leds/LedBaseCMD.java [hunk 2]
```diff
@@ -16,4 +16,7 @@ protected LedBaseCMD(Led2023 ledStrip) {
   public void initialize() {
     ledStrip.resetTimers();
   }
+
+  @Override
+  public void execute() {}
```
Comment (john-petrangelo) [inline src/main/java/frc/robot/commands/leds/LedBaseCMD.java:21]: Is there a reason to add an empty execute() function here?

File: src/main/java/frc/robot/constants/CompBotConstants.java [hunk 3]
```diff
@@ -138,7 +138,7 @@ public int intakeCubeLimitSwitchID() {
 
   @Override
   public int ledChannel() {
-    return 0;
+    return 4;
```
Comment (john-petrangelo) [inline src/main/java/frc/robot/constants/CompBotConstants.java:141]: ??

File: src/main/java/frc/robot/subsystems/led/Led2023.java [hunk 4]
```diff
@@ -113,145 +122,229 @@ public Led2023(Arm arm, IntakeRelease intakerelease) {
...
+        && DriverStation.isTeleopEnabled()
+        && !FINISHED_RAINBOW_ONCE) {
+      defaultTimer.start();
+      return ColorScheme.DEFAULT;
+    }
+    if (defaultTimer.hasElapsed(5) && !FINISHED_RAINBOW_ONCE) {
+      defaultTimer.reset();
+      defaultTimer.stop();
+      FINISHED_RAINBOW_ONCE = true;
+    }
```
Comment (john-petrangelo) [inline src/main/java/frc/robot/subsystems/led/Led2023.java:169]: Should this have returned something? Or is it ok to continue?

**Review by john-petrangelo — APPROVED** [review]
I'm approving, but look at my comments.

**Review by AdamCraftmaster — APPROVED** [review]
lgtm
